pkgs/applications/networking/cluster/kyverno-chainsaw/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

Comment:
This `default.nix` file should be moved to `pkgs/by-name/ky/kyverno-chainsaw.package.nix`

Reply:
`default.nix` file should be moved to `pkgs/applications/networking/cluster/kyverno-chainsaw/kyverno-chainsaw.package.nix` is this correct